src/devices/SM28BYJ48/SM28BYJ48.cs

```diff
@@ -0,0 +1,202 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Gpio;
+using System.Threading;
+
+namespace Iot.Device.SM28BYJ48
+{
+    /// <summary>
+    /// Can move the motor clockwise and counterclockwise.
+    /// </summary>
+    public enum RotationDirection
```

Comment:
One file per type (in VS you can use Ctrl-. to extract it automatically)

Reply:
Moved. Thanks for the useful hotkey :-)